src/zc/buildout/easy_install.py

```diff
@@ -213,20 +233,34 @@ def __init__(self,
 
         self._index_url = index
         path = (path and path[:] or []) + buildout_and_setuptools_path
-        if dest is not None and dest not in path:
-            path.insert(0, dest)
         self._path = path
         if self._dest is None:
             newest = False
         self._newest = newest
-        self._env = pkg_resources.Environment(path)
         self._index = _get_index(index, links, self._allow_hosts)
         self._requirements_and_constraints = []
         self._check_picked = check_picked
 
         if versions is not None:
             self._versions = normalize_versions(versions)
 
+    @reify
+    def _env(self):
```

Comment:
Why does this have to be lazy?

Why not just cale this make_env and call it from ``__iniit__``?